### ISSUE TITLE
[IMPROVEMENT] Need Build Instructions + Better Build Method

### ISSUE THREAD

**Issue by dabockster** [marticliment/UniGetUI#3040]
### Please confirm these before moving forward.

- [X] I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This improvement refers to an existing feature. If you want to suggest a new feature, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29).
- [X] This improvement is not a bug. If you want to report a bug, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=bug&projects=&template=bug-issue.yml&title=%5BBUG%5D+%28Enter+your+description+here%29).

### Describe the improvement

There's build_release.bat in the main dir of the repo, but there's no instructions on anything beyond that. There's no information on prerequisite libraries required, no manual compilation steps, nothing. Just a batch file that executes a ton of nested Python code. Having an actual build method in like Clang or something would be way better.

### Describe how this improvement could help users

The current build method executes a bunch of nested Python code. There's no way to tell what that code is doing at a glance. Having a standard build method with detailed instructions ensures that the build can be reproduced, is secure, and can be contributed to in the future (you know, because open source).

**Comment by marticliment** [comment]
> There's build_release.bat in the main dir of the repo, but there's no instructions on anything beyond that. There's no information on prerequisite libraries required, no manual compilation steps, nothing. Just a batch file that executes a ton of nested Python code. Having an actual build method in like Clang or something would be way better.

This is used to build the final build, with the installed.
For test builds, visual studio or jetbrains rider should be plug and play. Just open the solution and the editor should figure everything out, since UniGetUI is built with .NET projects

**Comment by marticliment** [comment]
However, I agree this needs clarification

**Comment by mrixner** [comment]
There's an [instructions page on your website](https://www.marticliment.com/unigetui/help/build-overview/), it just requires a lot of digging to find (and appears to be for old WingetUI, that ran on Python).

**Comment by marticliment** [comment]
yes, documentation on my web page is quite outdated, I have to work on it...
